beacon-chain/state/state-native/getters_validator.go

```diff
@@ -3,55 +3,40 @@ package state_native
 import (
 	"fmt"
 
-	"github.com/pkg/errors"
 	"github.com/prysmaticlabs/prysm/v4/beacon-chain/state"
+	"github.com/prysmaticlabs/prysm/v4/config/features"
 	fieldparams "github.com/prysmaticlabs/prysm/v4/config/fieldparams"
 	"github.com/prysmaticlabs/prysm/v4/consensus-types/primitives"
 	"github.com/prysmaticlabs/prysm/v4/encoding/bytesutil"
 	ethpb "github.com/prysmaticlabs/prysm/v4/proto/prysm/v1alpha1"
 	"github.com/prysmaticlabs/prysm/v4/runtime/version"
 )
 
-// ValidatorIndexOutOfRangeError represents an error scenario where a validator does not exist
-// at a given index in the validator's array.
-type ValidatorIndexOutOfRangeError struct {
```

Comment:
Removing this in favor of a unified `index %d out of bounds` message which is returned from the multi-value slice.